android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/BrightnessModule.java

```diff
@@ -45,18 +43,46 @@ public void run() {
   @ReactMethod
   public void getBrightnessAsync(final Promise promise) {
     WindowManager.LayoutParams lp = getCurrentActivity().getWindow().getAttributes();
```

Comment:
Reading through the implementation of Window and LayoutParams, it looks like the code isn't thread-safe. It won't crash but the LayoutParams object is mutable so I don't think we want to share it across multiple threads. So scheduling a UI thread Runnable seems like the best way to ensure mutual exclusion.